setup.py

```diff
@@ -26,6 +26,7 @@
 except ImportError:
     pass
 
-setuptools.setup(
-    setup_requires=['pbr>=1.8'],
-    pbr=True)
+setuptools.setup(setup_requires=['pbr>=1.8'],
+                 pbr=True,
+                 test_suite='test',
```

Comment:
Allows `python3 setup.py test`, that's a neat way to run the tests locally without tox.